bentoml/_internal/yatai_rest_api_client/schemas.py

```diff
@@ -24,7 +23,7 @@ def datetime_encoder(time_obj: Optional[datetime]) -> Optional[str]:
 def datetime_decoder(datetime_str: Optional[str], _) -> Optional[datetime]:
     if not datetime_str:
         return None
-    return parse(datetime_str)
+    return datetime.strptime(datetime_str, time_format)
```

Comment:
@parano @yetone was there a reason you were using `dateutil` for this?

Reply:
@sauyon Because the default JSON time format in Go is RFC 3339

https://pkg.go.dev/time#Time.MarshalJSON

![image](https://user-images.githubusercontent.com/1206493/179671008-63925e93-4ada-4a91-8e56-28223e4842ee.png)

Reply:
Should we use that format when saving too then?

Reply:
`datetime.strptime` should be RFC3339 compatible right?

Reply:
Maybe I'll just revert this change for now anyway; we can worry about this later.

Reply:
Maybe I'll just revert this change for now anyway; we can worry about this later.

Reply:
Let's just revert this :)